pkg/components/velero/component_test.go

```diff
@@ -50,24 +52,71 @@ component "velero" {
   }
 }
 `
+	configHCLOpenEBS := `
+component "velero" {
+  provider = "openebs"
+  openebs {
+    credentials  = <<EOF
+[default]
+access_key = "access_key"
+secret_key = "secret_key"
+EOF
 
-	component := newComponent()
+    volume_snapshot_location {
+      bucket           = "test_bucket"
+      provider         = "aws"
+      region           = "eu-west-1"
+    }
 
-	body, diagnostics := util.GetComponentBody(configHCL, name)
-	if diagnostics != nil {
-		t.Fatalf("Error getting component body: %v", diagnostics)
-	}
+    backup_storage_location {
+      bucket           = "test_bucket"
+      provider         = "aws"
+      region           = "eu-west-1"
+    }
+  }
+}
+`
+	configHCLRestic := `
+component "velero" {
+  provider = "restic"
+  restic {
+    credentials  = <<EOF
+[default]
+access_key = "access_key"
+secret_key = "secret_key"
+EOF
 
-	diagnostics = component.LoadConfig(body, &hcl.EvalContext{})
-	if diagnostics.HasErrors() {
-		t.Fatalf("Valid config should not return error, got: %s", diagnostics)
-	}
+    backup_storage_location {
+      bucket           = "test_bucket"
+      provider         = "aws"
+    }
+  }
+}
+`
 
-	m, err := component.RenderManifests()
-	if err != nil {
-		t.Fatalf("Rendering manifests with valid config should succeed, got: %s", err)
-	}
-	if len(m) <= 0 {
-		t.Fatalf("Rendered manifests shouldn't be empty")
+	configHCLs := []string{configHCLAzure, configHCLOpenEBS, configHCLRestic}
+	for _, configHCL := range configHCLs {
+		component := newComponent()
```

Comment:
We could use `components.Get()`, then we can make this file be part of `velero_test` package.